pkg/restapi/models/models.go

```diff
@@ -19,6 +19,21 @@ type DataVaultConfiguration struct {
 	HMAC        IDTypePair `json:"hmac"`
 }
 
+// DataVaultConfigurationMapping represents an entry in the data vault config store
+// It allows mapping from a vaultID + couchdbVaultName pair to a DataVaultConfiguration
+type DataVaultConfigurationMapping struct {
+	DataVaultConfiguration DataVaultConfiguration `json:"dataVaultConfiguration"`
+	VaultIDDbVaultNamePair VaultIDDbVaultNamePair `json:"vaultIDMapping"`
+}
+
+// VaultIDDbVaultNamePair represents a vaultID + couchDBStoreName pair.
+// VaultID is a base58-encoded random value that is returned as the vault id, and CouchDBVaultName is a raw uuid used
+// as the database name in couchdb to satisfy couchdb's database naming requirements
+type VaultIDDbVaultNamePair struct {
+	VaultID          string `json:"vaultId"`
+	CouchDBVaultName string `json:"couchdbVaultName"`
+}
+
```

Comment:
CouchDB-specific things should go with the CouchDBEDVStore stuff

Reply:
done!